apps/web/components/booking/BookingListItem.tsx

```diff
@@ -154,6 +165,37 @@ function BookingListItem(booking: BookingItemProps) {
 
   const startTime = dayjs(booking.startTime).format(isUpcoming ? "ddd, D MMM" : "D MMMM YYYY");
   const [isOpenRescheduleDialog, setIsOpenRescheduleDialog] = useState(false);
+  const [isOpenSetLocationDialog, setIsOpenLocationDialog] = useState(false);
+
+  const setLocationMutation = useMutation(async (newLocation: string) => {
+    const result = await fetch("/api/book/set-location", {
```

Comment:
renamed setLocation to set-location, urls are better with dash to separate 2 words